input/templates/partials/form/choose-country.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="{{containerClass}} input-block">
   <span class="text-uppercase">{{i18n "main:form.country"}}*</span><br>
-  <select id="{{selectId}}" class="checkout-dropdown" name="{{selectName}}{{suffix}}">
+  <select id="{{selectId}}" class="checkout-dropdown" name="{{options.formName}}{{#unless options.formName}}{{selectName}}{{/unless}}">
```

Comment:
formName sounds like the name of the form, but this is rather the name of the form field, so formFieldName?